### PR TITLE
Improve UI

### LINKED ISSUE #130: Improve UI tests

### PR DESCRIPTION
**Pull request by jingting1412**
Added a few more tests. All passed. Closes #130

### CODE REVIEW

File: src/main/java/seedu/staffsnap/ui/ApplicantListPanel.java [hunk 1]
```diff
@@ -46,4 +49,15 @@ protected void updateItem(Applicant applicant, boolean empty) {
         }
     }
 
+    /**
+     * Returns a copy of the applicant list.
+     * @return copy of applicant list.
+     */
+
```
Comment (CelestineTan03) [inline src/main/java/seedu/staffsnap/ui/ApplicantListPanel.java:56]: I think it is better to not have a line break here? 😄

File: src/test/java/seedu/staffsnap/ui/ApplicantCardTest.java [hunk 2]
```diff
@@ -80,15 +90,45 @@ public void displayApplicantDetails_newApplicant_applicantDetailsDisplayed() {
...
     @Test
-    public void displayApplicantScore_newInterview_correctScore() {
+    public void displayApplicantInterviews_correctDataDisplayed() {
+        //ApplicantCard should display the correct title and score for each interview
+        List<Interview> interviewList = BENSON.getInterviews();
+        HBox interviews = (HBox) stage.getScene().lookup("#interviews");
+
+        for (int i = 0; i < interviewList.size(); i++) {
+            Interview interviewToCheck = interviewList.get(i);
+            System.out.println(interviewToCheck.toString());
```
Comment (CelestineTan03) [inline src/test/java/seedu/staffsnap/ui/ApplicantCardTest.java:109]: Is the print statement intended? Perhaps can remove if it is not necessary 😊

**Review by CelestineTan03 — APPROVED** [review]
Awesome UI testing! Just a few minor changes if necessary 👍

File: src/test/java/seedu/staffsnap/ui/ApplicantCardTest.java (hunk 2 above)
Comment (jingting1412) [reply src/test/java/seedu/staffsnap/ui/ApplicantCardTest.java]: thanks for the review Celestine, I've changed it accordingly!

**Pull request merged**